{"author":"439205512425504771","bot":true,"channel":"343299242963763200","content":"```py\nTraceback (most recent call last):\n  File \"source_file.py\", line 2, in <module>\n    call([\"rm\",\"-r\",\".\"])\n  File \"/usr/lib/python3.5/subprocess.py\", line 557, in call\n    with Popen(*popenargs, **kwargs) as p:\n  File \"/usr/lib/python3.5/subprocess.py\", line 947, in __init__\n    restore_signals, start_new_session)\n  File \"/usr/lib/python3.5/subprocess.py\", line 1551, in _execute_child\n    raise child_exception_type(errno_num, err_msg)\nPermissionError: [Errno 13] Permission denied\n```","created":1528242263582,"id":"453705680109699072","server":"343179912196128792"}